net/cifsd-tools/Makefile

```diff
@@ -18,28 +18,69 @@ PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
 PKG_REMOVE_FILES:=autogen.sh
 
+PKG_BUILD_DEPENDS:=glib2
+
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 
-define Package/cifsd-tools
+define Package/cifsd-tools/Default
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Filesystem
-  TITLE:=Kernel CIFS/SMB server support and userspace tools
+  TITLE:=Kernel SMB/CIFS
   URL:=https://github.com/cifsd-team/cifsd-tools
-  DEPENDS:=+kmod-fs-cifsd +glib2 +libnl-core +libnl-genl
+  DEPENDS:= +libnl-core +libnl-genl $(ICONV_DEPENDS) $(INTL_DEPENDS)
```

Comment:
libnl dependency should be added to cifsd-server, as cifsd-utils does not link to it.

Reply:
fixed, upstream needs to update there am files too